processing/data_collection/gazette/spiders/rj_belford_roxo.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+import re
+
+import dateparser
+from scrapy import Request
+from scrapy.exceptions import CloseSpider
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjBelfordRoxoSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3300456"
+
+    allowed_domains = ["prefeituradebelfordroxo.rj.gov.br"]
+    name = "rj_belford_roxo"
+    start_urls = ["https://prefeituradebelfordroxo.rj.gov.br/atos-oficiais/"]
+
+    def parse(self, response):
+        for link in response.xpath(
+            '//article[contains(@class, "post-listing")]//div[@class="entry"]/p/a'
+        ):
+            date = self.get_date(link)
```

Comment:
**issue**: We need to limit as much as possible the number of requests we made. Each spider has a `self.start_date` argument that can be sent to filter the gazettes we want to gather. Use that to limit the gazettes you will return.